### ISSUE TITLE
Provisioning failing on db restart

### ISSUE THREAD

**Issue by jschider** [jsha/blocktogether#271]
Hello,

This is similar to #210, I'm on e71d154 (latest) and getting an error when mysql is restarting.  I just cloned this morning and have been trying various things to get it running.  I was able to clone release v1.7.2, but then the run-dev.sh failed.

`default: + sudo mkdir -p /data/
    default: + sudo mv /var/lib/mysql /data/mysql
    default: + sudo mkdir /data/mysql/tmp -p
    default: + sudo chown mysql.mysql /data/mysql/tmp
    default: + sudo mkdir /var/lib/mysql
    default: + sudo chown mysql.mysql /var/lib/mysql
    default: + sudo service mysql restart
    default: Job for mariadb.service failed because the control process exited with error code.
    default: See "systemctl status mariadb.service" and "journalctl -xe" for details.
#<Thread:0x000055a2ba0ef9e0@/usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/batch_action.rb:71 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
	109: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'`

The systemctl: 
`vagrant@ubuntu-bionic:~$ systemctl status mariadb.service
● mariadb.service - MariaDB 10.1.38 database server
   Loaded: loaded (/lib/systemd/system/mariadb.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Mon 2019-03-04 20:12:27 UTC; 15min ago
     Docs: man:mysqld(8)
           https://mariadb.com/kb/en/library/systemd/
  Process: 10779 ExecStart=/usr/sbin/mysqld $MYSQLD_OPTS $_WSREP_NEW_CLUSTER $_WSREP_START_POSITION (code=exited, status=1/FAILURE)
  Process: 10676 ExecStartPre=/bin/sh -c [ ! -e /usr/bin/galera_recovery ] && VAR= ||   VAR=`/usr/bin/galera_recovery`; [ $? -eq 0 ]   && syst
  Process: 10667 ExecStartPre=/bin/sh -c systemctl unset-environment _WSREP_START_POSITION (code=exited, status=0/SUCCESS)
  Process: 10645 ExecStartPre=/usr/bin/install -m 755 -o mysql -g root -d /var/run/mysqld (code=exited, status=0/SUCCESS)
 Main PID: 10779 (code=exited, status=1/FAILURE)
lines 1-10/10 (END)
`

**Comment by jsha** [comment]
The 1.7.2 tag is very old (2015). I haven't been tagging releases lately. Have you tried master?

**Comment by jschider** [comment]
Yes, I did try master several times and went through the commit logs.  I did see a couple commits ago changes near that section.  I did not try pulling down that version yet.

**Comment by jschider** [comment]
I did switch to the commit 7837f6f from December 4th and it loaded up the trusty32 and maria-db just fine.  I was able to run it locally as well.  It looks like the commit on December 30th is the culprit.
